### PR TITLE
Add CRD generate-check test to cluster-baremetal-operator

### PR DESCRIPTION
**Pull request by kirankt**
Add CRD generate-check test to cluster-baremetal-operator

### CODE REVIEW

File: ci-operator/config/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-master.yaml [hunk 1]
```diff
@@ -49,6 +49,10 @@ tests:
   commands: make vet
   container:
     from: src
+- as: crd-gen-check
```
Comment (dhellmann) [inline ci-operator/config/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-master.yaml:52]: It's going to check all of the generated output, right? Maybe a better name would be just "generate-check"?

**Review by sadasu — COMMENTED** [review]
/lgtm

**Pull request merged**